src/raw_logging.cc

```diff
@@ -120,16 +109,13 @@ void RawLog__(LogSeverity severity, const char* file, int line,
     return;  // this stderr log message is suppressed
   }
   // can't call localtime_r here: it can allocate
-  struct ::tm& t = last_tm_time_for_raw_log;
   char buffer[kLogBufSize];
   char* buf = buffer;
   int size = sizeof(buffer);
 
   // NOTE: this format should match the specification in base/logging.h
-  DoRawLog(&buf, &size, "%c%02d%02d %02d:%02d:%02d.%06d %5u %s:%d] RAW: ",
+  DoRawLog(&buf, &size, "%c0000 00:00:00.000000 %5u %s:%d] RAW: ",
```

Comment:
Ukai-san, could you check if the internal version has these numbers? If the internal version has removed them, we should do the same, too.

Reply:
it uses abseil-cpp's raw_logging

https://github.com/abseil/abseil-cpp/blob/master/absl/base/internal/raw_logging.cc